integration-test/src/main/java/com/sequenceiq/it/util/TagsUtil.java

```diff
@@ -134,31 +142,39 @@ private void validateClouderaCreatorResourceNameTag(TaggedResponse response, Str
         if (StringUtils.isEmpty(tagValue)) {
             tagValue = response.getTagValue(tag.toLowerCase());
         }
-        if (gcpLabelTransformedValue(tagValue, actingUserCrn.toString())) {
+        if (actingUserCrn != null && gcpLabelTransformedValue(tagValue, actingUserCrn.toString())) {
             Log.log(LOGGER, format(" Default tag: [%s] value is: [%s] equals [%s] acting user CRN transformed to a GCP label value! ", tag, tagValue,
                     actingUserCrn));
         } else {
```

Comment:
actingUserCrn can be null in the else and it could case NPE in crnEqualsWithoutConsideringPartition

Reply:
i think it is ensured that actingusercrn won't be null

Reply:
I fixed it, as fixed many condition with null and empty string check :)